### PR TITLE
feature/AT-SCE-UI_SaveOperation

### PR DESCRIPTION
**Pull request by jaimeIxba**
We add changes to the files "at-sce.controller.js", "at-sce-util.js", "calculator.hbs" and "at-sce-api.service.js" to integrate the POST method (create history).

### CODE REVIEW

File: src/config/passport.js [hunk 1]
```diff
@@ -11,7 +11,6 @@
 // Constants
```
Comment (at-internship) [inline src/config/passport.js:11]: what is the change on this file?

File: src/controllers/at-sce.controller.js [hunk 2]
```diff
@@ -48,9 +51,127 @@ atSCEController.calculator = async (req, res) => {
...
+      taxIva_r,
+      totalTaxes,
+      totalRevenue,
+    } = req.body;
+    const userErrors = [];
+    console.log(req.body);
+
+    // Validations
+    if (!rent) {
+      userErrors.push({ text: "Empty Field. " });
```
Comment (at-internship) [inline src/controllers/at-sce.controller.js]: Try to add more descriptive message

File: src/controllers/at-sce.controller.js [hunk 3]
```diff
@@ -48,9 +51,127 @@ atSCEController.calculator = async (req, res) => {
...
+    let request = {
+      type: projectType,
+      user_id: user_id,
+      fixedExpenses: {
+        rent: parseFloat(rent).toFixed(2),
+        transport: parseFloat(transport).toFixed(2),
+        internet: parseFloat(telephone).toFixed(2),
+        feed: parseFloat(feeding).toFixed(2),
+        others: parseFloat(others).toFixed(2),
+        total: total,
```
Comment (at-internship) [inline src/controllers/at-sce.controller.js]: don't you think we also need parseFloat here?

**Review by at-internship — CHANGES_REQUESTED** [review]

**Review by DianaArceoSuarez — APPROVED** [review]

**Review by at-internship — APPROVED** [review]
Approved

**Review by FlavioAlejandro — APPROVED** [review]

**Review by AT-ivillamil — APPROVED** [review]

**Pull request merged**